src/components/modes/teacher/BackgroundToggle.tsx

```diff
@@ -84,91 +72,25 @@ const BackgroundToggle = (): JSX.Element => {
     }
   };
 
-  const handleScaleChange = (
-    event: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>,
-  ): void => {
-    let newScaleFloat = 1.0;
-    try {
-      newScaleFloat = parseFloat(event.target?.value);
-      if (typeof newScaleFloat !== 'number') return;
-    } catch (error) {
-      setScaleError(true);
-      return;
-    }
-    setBackgroundScale(newScaleFloat);
-    setScaleError(false);
-    if (newScaleFloat > 0.0001) {
-      if (backgroundSettings?.id) {
-        const newBackgroundSettings = {
-          ...backgroundSettings,
-          data: {
-            ...backgroundSettings.data,
-            scale: newScaleFloat,
-          },
-        };
-        patchAppSetting(newBackgroundSettings);
-      } else {
-        postAppSetting({
-          ...DEFAULT_BACKGROUND_SETTINGS,
-          data: {
-            scale: newScaleFloat,
-          },
-        });
-      }
-    } else {
-      setScaleError(true);
-    }
-  };
-
   return (
-    <>
-      <ToggleContainer>
-        <Typography
-          fontSize="1.1em"
-          color={toggleDisabled ? grey[500] : 'black'}
-        >
-          {t('Show Background Image')}
-        </Typography>
-        <FormControlLabel
-          disabled={toggleDisabled}
-          control={
-            <Switch
-              color="primary"
-              checked={
-                backgroundSettings?.data?.toggle ?? DEFAULT_BACKGROUND_ENABLED
-              }
-              onChange={handleToggle}
-            />
-          }
-          label={undefined}
-        />
-      </ToggleContainer>
-      <ToggleContainer>
-        <Typography
-          fontSize="1.1em"
-          color={toggleDisabled ? grey[500] : 'black'}
-        >
-          {t('Scale background image')}
-        </Typography>
-        <TextField
-          disabled={toggleDisabled}
-          value={backgroundScale}
-          error={scaleError}
-          color="primary"
-          id="outlined-number"
-          type="number"
-          InputLabelProps={{
-            shrink: true,
-          }}
-          inputProps={{
-            max: 40,
-            min: 0.1,
-            step: 0.1,
-          }}
-          onChange={handleScaleChange}
-        />
-      </ToggleContainer>
-    </>
+    <ToggleContainer>
+      <Typography fontSize="1.1em" color={toggleDisabled ? grey[500] : 'black'}>
+        {t('Show Background Image')}
+      </Typography>
+      <FormControlLabel
```

Comment:
Is there a reason why you use the `FormControlLabel` with a label of undefined ? Would `FormControl` be more suitable for your use-case ?

Reply:
I think this is partially updated legacy code. I completely removed the form control label.